components/brave_wallet_ui/page/screens/swap/components/swap/header/header.tsx

```diff
@@ -142,7 +134,7 @@ export const Header = () => {
             hasBackground={true}
             hasShadow={true}
             isHeader={true}
-            networkNotSupported={!isNetworkSupported}
+            networkNotSupported={false}
```

Comment:
Do we need this prop anymore?

Reply:
Nope. If you're okay with me removing the associated React code, I can get rid of the prop.

Reply:
removed here: https://github.com/brave/brave-core/pull/20923/commits/a8fea1d18cfc48a7ba897f97414d7c93f2b10aa9